code/datums/gamemodes/distress.dm

```diff
@@ -37,12 +37,12 @@
 	for(var/obj/effect/landmark/corpsespawner/corpse AS in GLOB.corpse_landmarks_list)
 		corpse.create_mob()
 
-	for var/xenomorph/beo AS in GLOB.alive_xeno_list
-		if(isxenolarva(i)) // Larva
-			var/mob/living/carbon/xenomorph/larva/X = i
+	for(var/mob/living/carbon/xenomorph/xeno AS in GLOB.alive_xeno_list)
+		if(isxenolarva(xeno)) // Larva
+			var/mob/living/carbon/xenomorph/larva/X = xeno
```

Comment:
this and line 45 you can cut since the var is already typed now

Reply:
Done